server/applicationPattern/applicationPattern/onfModel/models/layerProtocols/HttpClientInterface.js

```diff
@@ -85,6 +86,21 @@ class HttpClientInterface extends layerProtocol {
         return httpClientConfiguration[onfAttributes.HTTP_CLIENT.APPLICATION_NAME];
     }
 
+    static async getHttpClientUuidFromForwarding (forwardingName) {
+        try {
+            let forwardConstructName = await ForwardingDomain.getForwardingConstructForTheForwardingNameAsync(forwardingName)
+            if (forwardConstructName === undefined) {
+                return;
+            }
+            let forwardConstructUuid = forwardConstructName[onfAttributes.GLOBAL_CLASS.UUID]
+            let fcPortOutput = (await ForwardingConstruct.getOutputFcPortsAsync(forwardConstructUuid))[0]
+            let operationClientUuid = fcPortOutput[onfAttributes.FC_PORT.LOGICAL_TERMINATION_POINT];
+            let httpClientUuid = (await logicalTerminationPoint.getServerLtpListAsync(operationClientUuid))[0];
+            return httpClientUuid;
+        } catch (error) {
+            console.log(error);
```

Comment:
do not hide exceptions, remove such try/catch

Reply:
Thank you for indicating .